Failure running HistoricalAccountInfoMigration when run as a jar

**Detailed Description**
Failure running v0.29.0 with java -jar during HistoricalAccountInfoMigration.

**Actual Behavior**
```
Mar 23 22:50:27 coinnode-hedera-mainnet-SFO2-01 java[3208127]: 2021-03-23T16:50:27.899-0600 INFO main c.h.m.i.m.HistoricalAccountInfoMigration Importing historical account information
Mar 23 22:50:27 coinnode-hedera-mainnet-SFO2-01 java[3208127]: 2021-03-23T16:50:27.905-0600 ERROR main o.f.c.i.c.DbMigrate Migration of schema "public" with repeatable migration "Import historical account information from before open access" failed! Changes successfully rolled back.
Mar 23 22:50:27 coinnode-hedera-mainnet-SFO2-01 java[3208127]: 2021-03-23T16:50:27.910-0600 WARN main o.s.b.w.r.c.AnnotationConfigReactiveWebServerApplicationContext Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: Migration failed !
Mar 23 22:50:28 coinnode-hedera-mainnet-SFO2-01 java[3208127]: 2021-03-23T16:50:28.023-0600 ERROR main o.s.b.SpringApplication Application run failed org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: Migration failed !\\n        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1786)\\n        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:602)\\n        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:524)\\n        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)\\n        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)\\n        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)\\n        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)\\n        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)\\n        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)\\n        at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1153)\\n        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:907)\\n        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:582)\\n        at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.refresh(ReactiveWebServerApplicationContext.java:63)\\n        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:767)\\n        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:759)\\n        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:426)\\n        at org.springframework.boot.SpringApplication.run(SpringApplication.java:326)\\n        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1311)\\n        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1300)\\n        at com.hedera.mirror.importer.MirrorImporterApplication.main(MirrorImporterApplication.java:32)\\n        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\\n        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\\n        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\\n        at java.base/java.lang.reflect.Method.invoke(Method.java:566)\\n        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)\\n        at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)\\n        at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)\\n        at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)\\nCaused by: org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: Migration failed !\\n        at org.flywaydb.core.internal.command.DbMigrate.doMigrateGroup(DbMigrate.java:438)\\n        at org.flywaydb.core.internal.command.DbMigrate.access$200(DbMigrate.java:56)\\n        at org.flywaydb.core.internal.command.DbMigrate$3.call(DbMigrate.java:341)\\n        at org.flywaydb.core.internal.jdbc.TransactionalExecutionTemplate.execute(TransactionalExecutionTemplate.java:66)\\n        at org.flywaydb.core.internal.command.DbMigrate.applyMigrations(DbMigrate.java:338)\\n        at org.flywaydb.core.internal.command.DbMigrate.migrateGroup(DbMigrate.java:303)\\n        at org.flywaydb.core.internal.command.DbMigrate.access$100(DbMigrate.java:56)\\n        at org.flywaydb.core.internal.command.DbMigrate$2.call(DbMigrate.java:207)\\n        at org.flywaydb.core.internal.command.DbMigrate$2.call(DbMigrate.java:204)\\n        at org.flywaydb.core.internal.database.postgresql.PostgreSQLAdvisoryLockTemplate.execute(PostgreSQLAdvisoryLockTemplate.java:73)\\n        at org.flywaydb.core.internal.database.postgresql.PostgreSQLConnection.lock(PostgreSQLConnection.java:99)\\n        at org.flywaydb.core.internal.schemahistory.JdbcTableSchemaHistory.lock(JdbcTableSchemaHistory.java:139)\\n        at org.flywaydb.core.internal.command.DbMigrate.migrateAll(DbMigrate.java:204)\\n        at org.flywaydb.core.internal.command.DbMigrate.migrate(DbMigrate.java:165)\\n        at org.flywaydb.core.Flyway$1.execute(Flyway.java:215)\\n        at org.flywaydb.core.Flyway$1.execute(Flyway.java:164)\\n        at org.flywaydb.core.Flyway.execute(Flyway.java:538)\\n        at org.flywaydb.core.Flyway.migrate(Flyway.java:164)\\n        at org.springframework.boot.autoconfigure.flyway.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:66)\\n        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1845)\\n        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1782)\\n        ... 27 more\\nCaused by: org.flywaydb.core.api.FlywayException: Migration failed !\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor.executeOnce(JavaMigrationExecutor.java:79)\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor.access$000(JavaMigrationExecutor.java:34)\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor$1.call(JavaMigrationExecutor.java:57)\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor$1.call(JavaMigrationExecutor.java:54)\\n        at org.flywaydb.core.internal.database.DefaultExecutionStrategy.execute(DefaultExecutionStrategy.java:28)\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor.execute(JavaMigrationExecutor.java:54)\\n        at org.flywaydb.core.internal.command.DbMigrate.doMigrateGroup(DbMigrate.java:434)\\n        ... 47 more\\nCaused by: java.io.FileNotFoundException: class path resource [accountInfo.txt.gz] cannot be resolved to absolute file path because it does not reside in the file system: jar:file:/opt/hedera/hedera-mirror-importer-v0.29.0.jar!/BOOT-INF/classes!/accountInfo.txt.gz\\n        at org.springframework.util.ResourceUtils.getFile(ResourceUtils.java:217)\\n        at org.springframework.core.io.AbstractFileResolvingResource.getFile(AbstractFileResolvingResource.java:154)\\n        at com.hedera.mirror.importer.migration.HistoricalAccountInfoMigration.doMigrate(HistoricalAccountInfoMigration.java:106)\\n        at com.hedera.mirror.importer.migration.MirrorBaseJavaMigration.migrate(MirrorBaseJavaMigration.java:45)\\n        at org.flywaydb.core.internal.resolver.java.JavaMigrationExecutor.executeOnce(JavaMigrationExecutor.java:65)\\n        ... 53 more
```

**Expected Behavior**
Migration to complete

**Environment:**
- Java: OpenJDK 11.0
- OS: Ubuntu 18.04
- Version: v0.29.0

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

--- a/charts/hedera-mirror-common/Chart.yaml
+++ b/charts/hedera-mirror-common/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: Hedera Mirror Node common components installed globally for use across namespaces
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror-common
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror-grpc/Chart.yaml
+++ b/charts/hedera-mirror-grpc/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: GRPC API for the Hedera Mirror Node
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror-grpc
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror-importer/Chart.yaml
+++ b/charts/hedera-mirror-importer/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: Hedera Mirror Importer downloads and validates file streams from the cloud and imports them into the database
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror-importer
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror-monitor/Chart.yaml
+++ b/charts/hedera-mirror-monitor/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: End to End Monitor for the Hedera Mirror Node
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror-monitor
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror-rest/Chart.yaml
+++ b/charts/hedera-mirror-rest/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: REST API for the Hedera Mirror Node
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror-rest
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror/Chart.yaml
+++ b/charts/hedera-mirror/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.29.0
+appVersion: 0.29.1
 description: Hedera Mirror Node mirrors data from Hedera nodes and serves it via an API
 home: https://github.com/hashgraph/hedera-mirror-node
 icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
@@ -11,4 +11,4 @@ maintainers:
 name: hedera-mirror
 sources:
   - https://github.com/hashgraph/hedera-mirror-node
-version: 0.16.0
+version: 0.16.1

--- a/charts/hedera-mirror/requirements.lock
+++ b/charts/hedera-mirror/requirements.lock
@@ -1,13 +1,13 @@
 dependencies:
 - name: hedera-mirror-grpc
   repository: file://../hedera-mirror-grpc
-  version: 0.16.0
+  version: 0.16.1
 - name: hedera-mirror-importer
   repository: file://../hedera-mirror-importer
-  version: 0.16.0
+  version: 0.16.1
 - name: hedera-mirror-monitor
   repository: file://../hedera-mirror-monitor
-  version: 0.16.0
+  version: 0.16.1
 - name: postgresql-ha
   repository: https://charts.bitnami.com/bitnami
   version: 6.3.7
@@ -16,9 +16,9 @@ dependencies:
   version: 12.2.4
 - name: hedera-mirror-rest
   repository: file://../hedera-mirror-rest
-  version: 0.16.0
+  version: 0.16.1
 - name: timescaledb-multinode
   repository: https://raw.githubusercontent.com/timescale/timescaledb-kubernetes/master/charts/repo/
   version: 0.8.0
-digest: sha256:f51f300a98b7ff818e50c98ce0fed76a4454ee5ee099963e78afaaa092d49a20
-generated: "2021-03-23T13:27:50.173939-05:00"
+digest: sha256:03eecdaf877e1d11571955b64d6456f9213dcf353c0bb8ce9984cc601da620d1
+generated: "2021-03-23T18:47:40.179794-05:00"

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -17,7 +17,7 @@ services:
       - 5432:5432
 
   grpc:
-    image: gcr.io/mirrornode/hedera-mirror-grpc:0.29.0
+    image: gcr.io/mirrornode/hedera-mirror-grpc:0.29.1
     restart: unless-stopped
     environment:
       HEDERA_MIRROR_GRPC_DB_HOST: db
@@ -29,7 +29,7 @@ services:
       - 5600:5600
 
   importer:
-    image: gcr.io/mirrornode/hedera-mirror-importer:0.29.0
+    image: gcr.io/mirrornode/hedera-mirror-importer:0.29.1
     restart: unless-stopped
     environment:
       HEDERA_MIRROR_IMPORTER_DB_HOST: db
@@ -41,7 +41,7 @@ services:
   monitor:
     deploy:
       replicas: 0
-    image: gcr.io/mirrornode/hedera-mirror-monitor:0.29.0
+    image: gcr.io/mirrornode/hedera-mirror-monitor:0.29.1
     restart: unless-stopped
     environment:
       SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-monitor/
@@ -58,7 +58,7 @@ services:
       - 6379:6379
 
   rest:
-    image: gcr.io/mirrornode/hedera-mirror-rest:0.29.0
+    image: gcr.io/mirrornode/hedera-mirror-rest:0.29.1
     environment:
       HEDERA_MIRROR_REST_DB_HOST: db
     restart: unless-stopped
@@ -67,7 +67,7 @@ services:
       - 5551:5551
 
   rosetta:
-    image: gcr.io/mirrornode/hedera-mirror-rosetta:0.29.0
+    image: gcr.io/mirrornode/hedera-mirror-rosetta:0.29.1
     environment:
       HEDERA_MIRROR_ROSETTA_DB_HOST: db
     restart: unless-stopped

--- a/hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/HistoricalAccountInfoMigration.java
+++ b/hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/HistoricalAccountInfoMigration.java
@@ -23,9 +23,7 @@ package com.hedera.mirror.importer.migration;
 import com.google.common.base.Stopwatch;
 import com.hederahashgraph.api.proto.java.CryptoGetInfoResponse.AccountInfo;
 import java.io.BufferedReader;
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
 import java.time.Instant;
@@ -102,10 +100,8 @@ public class HistoricalAccountInfoMigration extends MirrorBaseJavaMigration {
         log.info("Importing historical account information");
         Stopwatch stopwatch = Stopwatch.createStarted();
 
-        try (
-                InputStream inputStream = new GZIPInputStream(new FileInputStream(accountInfoPath.getFile()));
-                BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8))
-        ) {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(new GZIPInputStream(accountInfoPath
+                .getInputStream()), StandardCharsets.UTF_8))) {
             long count = reader.lines()
                     .map(this::parse)
                     .filter(Objects::nonNull)

--- a/hedera-mirror-rest/api/v1/openapi.yml
+++ b/hedera-mirror-rest/api/v1/openapi.yml
@@ -550,7 +550,7 @@ tags:
     description: The tokens object represents the information associated with a token entity and returns a list of token information.
 info:
   title: Hedera Mirror Node REST API
-  version: 0.29.0
+  version: 0.29.1
   license:
     name: Apache-2.0
     url: 'https://www.apache.org/licenses/LICENSE-2.0.html'

--- a/hedera-mirror-rest/monitoring/monitor_apis/package.json
+++ b/hedera-mirror-rest/monitoring/monitor_apis/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hedera-mirror-monitor",
-  "version": "0.29.0",
+  "version": "0.29.1",
   "description": "Hedera Mirror Node Monitor",
   "main": "server.js",
   "private": true,

--- a/hedera-mirror-rest/package.json
+++ b/hedera-mirror-rest/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hedera-mirror-rest",
-  "version": "0.29.0",
+  "version": "0.29.1",
   "description": "Hedera Mirror Node REST API",
   "main": "server.js",
   "private": true,

--- a/hedera-mirror-rosetta/config/application.yml
+++ b/hedera-mirror-rosetta/config/application.yml
@@ -15,4 +15,4 @@ hedera:
       port: 5700
       realm: 0
       shard: 0
-      version: 0.29.0
+      version: 0.29.1

--- a/hedera-mirror-test/src/test/resources/k8s/hcs-perf-message-submit.yml
+++ b/hedera-mirror-test/src/test/resources/k8s/hcs-perf-message-submit.yml
@@ -41,7 +41,7 @@ spec:
                     app.kubernetes.io/name: test
       restartPolicy: Never
       containers:
-        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.0
+        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.1
           name: test
           env:
             - name: testProfile

--- a/hedera-mirror-test/src/test/resources/k8s/hcs-perf-publish-test.yml
+++ b/hedera-mirror-test/src/test/resources/k8s/hcs-perf-publish-test.yml
@@ -43,7 +43,7 @@ spec:
                     app.kubernetes.io/name: test
       restartPolicy: Never
       containers:
-        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.0
+        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.1
           name: test
           env:
             - name: testProfile

--- a/hedera-mirror-test/src/test/resources/k8s/hcs-perf-subscribe-test.yml
+++ b/hedera-mirror-test/src/test/resources/k8s/hcs-perf-subscribe-test.yml
@@ -46,7 +46,7 @@ spec:
                     app.kubernetes.io/name: test
       restartPolicy: Never
       containers:
-        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.0
+        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.1
           name: test
           env:
             - name: testProfile

--- a/hedera-mirror-test/src/test/resources/k8s/hts-perf-batch-publish-batch-validate.yml
+++ b/hedera-mirror-test/src/test/resources/k8s/hts-perf-batch-publish-batch-validate.yml
@@ -47,7 +47,7 @@ spec:
                     app.kubernetes.io/name: test
       restartPolicy: Never
       containers:
-        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.0
+        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.1
           name: test
           env:
             - name: testProfile

--- a/hedera-mirror-test/src/test/resources/k8s/hts-perf-publish-and-retrieve.yml
+++ b/hedera-mirror-test/src/test/resources/k8s/hts-perf-publish-and-retrieve.yml
@@ -46,7 +46,7 @@ spec:
                     app.kubernetes.io/name: test
       restartPolicy: Never
       containers:
-        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.0
+        - image: gcr.io/mirrornode/hedera-mirror-test:0.29.1
           name: test
           env:
             - name: testProfile

--- a/pom.xml
+++ b/pom.xml
@@ -74,8 +74,8 @@
         <micrometer-jvm-extras.version>0.2.2</micrometer-jvm-extras.version>
         <msgpack.version>0.8.22</msgpack.version>
         <protobuf.version>3.15.5</protobuf.version>
-        <release.version>0.29.0</release.version> <!-- Used to replace release versions in all files -->
-        <release.chartVersion>0.16.0</release.chartVersion>
+        <release.version>0.29.1</release.version> <!-- Used to replace release versions in all files -->
+        <release.chartVersion>0.16.1</release.chartVersion>
         <replacer.version>1.5.3</replacer.version>
         <!--suppress UnresolvedMavenProperty -->
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>

